lib/time/time.go

```diff
@@ -10,16 +15,54 @@ import (
 	"go.starlark.net/syntax"
 )
 
-// Module time is a Starlark module of time-related functions.
+// Module time is a Starlark module of time-related functions and constants.
+// The module defines the following functions:
+//
+//     from_timestamp(sec, nsec) - Converts the given Unix time corresponding to the total amount of seconds
+//                                 and nonoseconds since January 1, 1970 UTC into an object of type Time.
+//                                 The total amount of nonoseconds is optional.
+//                                 It is valid to pass nonoseconds outside the range [0, 999999999].
+//                                 Not all sec values have a corresponding time value. One such
+//                                 value is 1<<63-1 (the largest int64 value).
+//
+//     is_valid_timezone(loc) - Indicates whether the given name of location is valid or not.
+//                              Returns true if it is valid, false otherwise.
+//
+//     now() - Returns the current local time.
+//
+//     parse_duration(d) - Parses the given duration string.
+//                         A duration string is a possibly signed sequence of decimal numbers,
+//                         each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m".
+//                         Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+//
+//     parseTime(x, format, location) - Parses the given time string using a specific time format and location.
+//                                      The expected arguments are a time string (mandatory), a time format
+//                                      (optional, set to RFC3339 by default) and a name of location
+//                                      (optional set to UTC by default).
+//
+//     time(year, month, day, hour, minute, second, nanosecond, location) - Returns the Time corresponding to
+//	                                                                        yyyy-mm-dd hh:mm:ss + nsec nanoseconds
+//                                                                          in the appropriate zone for that time
+//                                                                          in the given location. All the parameters
+//                                                                          are optional.
+// The module also defines the following constants:
+//
+// nanosecond - A duration representing one nanosecond.
```

Comment:
Indent each line by at least three spaces so that it appears in godoc as a block quote.